website/content/en/highlights/2021-08-24-helm-vector-dev.md

```diff
@@ -0,0 +1,30 @@
+---
+date: "2021-08-24"
+title: "Introducing https://helm.vector.dev"
+description: "A new home for Vector's Helm charts"
+authors: ["spencergilbert"]
+pr_numbers: []
+release: "0.16.0"
+hide_on_release_notes: false
+badges:
+  type: "deprecation"
+  platforms: ["helm"]
+---
+
```

Comment:
I might lead with a description of the new repository and _then_ the deprecation of the old one. Something like:

```
Vector's new home for helm charts is at https://helm.vector.dev!

We made this change as part of migrating the helm charts from our old AWS S3 bucket hosting to Github Pages via [timberio/helm-charts](https://github.com/timberio/helm-charts). This new domain will also allow us to swap hosting in the future without any user impact.
```